### PR TITLE
Update supporting-tools.md

### PR DESCRIPTION
**Pull request by bamurtaugh**
Update with Codespaces property mentioned here: https://github.com/devcontainers/devcontainers.github.io/pull/5#pullrequestreview-985299782.

### CODE REVIEW

**Review by Chuxel — APPROVED** [review]

File: docs/specs/supporting-tools.md [hunk 1]
```diff
@@ -70,6 +70,23 @@ A [codespace](https://docs.github.com/en/codespaces/overview) is a development e
 #### Product specific properties
 GitHub Codespaces works with a growing number of tools and, where applicable, their `devcontainer.json` properties. For example, connecting the Codespaces web editor or VS Code enables the use of [VS Code properties](#visual-studio-code).
 
+If your Codespaces project needs additional permissions for other repositories, you can configure this through the `repositories` and `permissions` properties. You may learn more about this in the [Codespaces documentation](https://docs.github.com/en/codespaces/managing-your-codespaces/managing-repository-access-for-your-codespaces). As with other tools, Codespaces specific properties are placed within a `codespaces` namespace inside the `customizations` property.
+
+```json
```
Comment (joshspicer) [inline docs/specs/supporting-tools.md:75]:
```suggestion
```jsonc
```

File: docs/specs/supporting-tools.md [hunk 2]
```diff
@@ -70,6 +70,23 @@ A [codespace](https://docs.github.com/en/codespaces/overview) is a development e
...
+"customizations": {
+		// Configure properties specific to Codespaces.
+		"codespaces": {
+			"repositories": {
+				"my_org/my_repo": {
+					"permissions": {
+						"issues": "write"
```
Comment (joshspicer) [inline docs/specs/supporting-tools.md:82]: I think `contents` is going to be the most commonly used (only attribute i've used so far), so saving a click by having that attribute here :) )

```suggestion
						"issues": "write",
						"contents": "read"
```

File: docs/specs/supporting-tools.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bamurtaugh) [reply docs/specs/supporting-tools.md:75]: Thanks! Updated.

Comment (bamurtaugh) [reply docs/specs/supporting-tools.md:75]: Thanks, updated!

**Pull request merged**